AWSCognito.java

```diff
@@ -0,0 +1,15 @@
+package com.example.AWSCodeGuru.TestCode;
+
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.ayth.BasicAWSCredentials;
+
+static String myKeyId = "AKIAX742FUDUQXXXXXXX";
+static String mySecretKey = "MySecretKey";
+
+public static void main(String[] args){
+	AWSCredentials creds = getCreds(myKeyId, mySecretKey);
+}
+
+static AWSCredentials getCreds(String id, String key){

```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on tokens, ConstructorDeclaration expected instead